R/geoserver_tools.R

```diff
@@ -300,7 +300,7 @@ discover_nhdplus_id <- function(point = NULL, nldi_feature = NULL) {
   if (!is.null(point)) {
     coords = sf::st_coordinates(point)
     comid  = dataRetrieval::findNLDI(location = c(X = coords[1],
```

Comment:
findNLDI should accept an sf point now.

Reply:
Good call -- I'll leave this for now though.